assignments/assignment1.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title: Assignment 1 - Data Collection and Analysis
+nav_exclude: true
+---
+
+This assignment is released as a Python Notebook [at this link](assignment1.ipynb). Please read the instructions there and complete the parts marked `TODO`.
+**This assignment involves a data collection step that contains action items as early as Monday. So please start early.**
+
+You can use Google Colab to edit the Python notebook or run your local Jupyter server.
+
+Once the Python notebook is completed, follow these steps to submit:
+
+  1. Create a private repo called `assignment-1` following the repo naming convention.
```

Comment:
this should be username-assignment-1 or whatever the instructions are in assignments.md